src/main/java/com/plaid/client/response/PaymentGetResponse.java

```diff
@@ -0,0 +1,48 @@
+package com.plaid.client.response;
+
+import com.plaid.client.model.payment.Amount;
+
+import java.util.Date;
+
+public final class PaymentGetResponse extends BaseResponse {
+  private String paymentId;
```

Comment:
This and `src/main/java/com/plaid/client/model/payment/Payment.java` are pretty much just duplicates, are both needed?

Reply:
I believe both are needed because `PaymentGetResponse` extends the `BaseResponse` class and Java only allows inheriting from one class (from looking up briefly online).